monitoring/metrics.md

```diff
@@ -0,0 +1,163 @@
+## Requests coming to / of the API
+
+- [Prometheus Flask exporter](https://github.com/rycus86/prometheus_flask_exporter)
+  - metrics are configured via decorators, e.g. `@metrics.counter(..)`:
+  ```python
+  @app.route('/<item_type>')
+  @metrics.do_not_track()
+  @metrics.counter('invocation_by_type', 'Number of invocations by type',
+           labels={'item_type': lambda: request.view_args['type']})
+  def by_type(item_type):
+      pass  # only the counter is collected, not the default metrics
+  ```
+  - the metrics are by default exposed on the same Flask application on the /metrics endpoint,
+    this can be adjusted
+  - counters count invocations, other types (histogram, gauge, summary) collect metrics based on the
+    duration of those invocations
+  - labels can be defined (also using attributes of the request or the response)
+
+## Celery metrics
+
+- Celery needs to be configured to send events to the broker which the exporter will collect (via config/CLI)
+
+- [danihodovic exporter](https://github.com/danihodovic/celery-exporter)
+  - [metrics](https://github.com/danihodovic/celery-exporter#metrics):
+    - celery_task_sent_total
+    - celery_task_received_total
+    - celery_task_started_total
+    - celery_task_succeeded_total
+    - celery_task_failed_total
+    - celery_task_rejected_total
+    - celery_task_revoked_total
+    - celery_task_retried_total
+    - celery_worker_up
+    - celery_worker_tasks_active
```

Comment:
these seem mega useful! especially since we don't measure anything in fedora-messaging - we can easily get overwhelmed from it without us noticing